.env.example

```
@@ -0,0 +1,38 @@
+# ===========================================
+# Copy this to .env for local development
+# Add these to Render's Environment Variables for production
+
+# ============ REQUIRED ============
+GROQ_API_KEY=your-groq-api-key
+WIX_BASE_URL=https://your-site.wixstudio.com/your-site
+PORT=8000
+
+# ============ OPTIONAL ============
+ENVIRONMENT=production
+CONFIG_API_KEY=your-config-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONFIG_API_KEY key should go before the ENVIRONMENT key

(UnorderedKey)